SO_properties.py

```diff
@@ -202,12 +205,15 @@ def compute_basics(self):
             typearr[:] = ptype
             types.append(typearr)
             groupnr.append(self.data[ptype]["GroupNr_bound"])
+            s = np.ones(r.shape, dtype='float64')*self.softening_of_parttype[ptype]
```

Comment:
This could maybe be np.float64 instead of a string "float64"

Reply:
I've added a commit with this change in https://github.com/SWIFTSIM/SOAP/pull/71